koji_containerbuild/plugins/hub_containerbuild.py

```diff
@@ -53,6 +58,30 @@ def _get_task_opts_and_opts(opts, priority, channel):
 
     return opts, taskOpts
 
+
+def _check_priority_allowlist(target, priority):
+    CONFIG = koji.read_config_files([(CONFIG_FILE, True)])
+    priority_target_list = CONFIG.get('priority_allowlist', 'priority_target_list_url')
+    req = Request(priority_target_list)
+    try:
+        response = urlopen(req)
```

Comment:
Is there any read time limit? I.e not waiting 2 minutes for TCP timeout, etc...

Reply:
I'll check

Reply:
Apparently only for Py 3. Another option is switching to `requests`.

Reply:
Fixed using `socket`'s global timeout. Works for 2 & 3.